mixins/js/ez-tabs.js

```diff
@@ -1,5 +1,5 @@
 window.eZ = window.eZ || {};
-
+/* jshint esversion:6 */
```

Comment:
that's not needed normally, see https://github.com/ezsystems/hybrid-platform-ui-core-components/blob/master/.eslintrc.json#L8